src/runtime/crdt/tests/crdt-entity-test.ts

```diff
@@ -39,7 +39,7 @@ describe('CRDTEntity', () => {
     };
     const entity = new CRDTEntity(singletons, collections);
 
-    const value = {id: 'bob', value: 'bob'} as Referenceable;
```

Comment:
So, `Referenceable` is still a thing in `crdt-collection.ts`, it seems like we are actually intending to define and use these as that. Am I off base? Is there an alternate way to define these so that they can actually be `Referenceable`? Maybe it just doesn't matter because this is only an issue here in the test?

Reply:
Off base, yeah. crdt-collection uses something that extends Referenceable.

Reply:
I agree. The problem here is that 'as' is an unsafe cast (in Typescript).
We should be using a `const value: <SomeType> = ...` but I'm not sure if there's an intermediate type there or we can just use Referenceable.